blog/2020-09-angular-schematics-common-helpers.md

```diff
@@ -682,11 +685,139 @@ export function playground(_options: any): Rule {
 }
 ```
 
-If we now run `schematics ../playground/src/collection.json:ng-add --debug=false` from our example Angular project, we can see that the `ng add` schematic has called the `playground` schematic.
+If we now run `schematics ../playground/src/collection.json:ng-add --debug=false` from our example Angular project, we can see that the `ng-add` schematic has called the `playground` schematic.
+
+With this knowledge you can define small atomic Schematics that can be executed 'standalone' or from another schematic that combines multiple standalone Schematics and calls them with specific parameters.
```

Comment:
"standalone" with Double quotes?

Reply:
Arent't single quotes the right one in English language here? My spell checker plugin suggested single quotes also.

Reply:
Piece from thre Angular docs:

> The exponentialStrength pipe executes every time the user changes the "normal power" value or the "boost factor", as shown in Figure 3.

https://angular.io/guide/pipes